### ISSUE TITLE
Issue with link contrast checking

### ISSUE THREAD

**Issue by ovann86** [inclusive-design/AChecker#63]
I believe I've come across an issue with the achecker that results in invalid colour contrasts being passed.

In the example attached we have:

- paragraph standard colour: #717171
- paragraph light colour: #0092c0
- link light colour: #0092c0

When you run it through the achecker test against WCAG 2.0 AA - only the paragraph light colour element comes up with an error.

From my understanding of the requirements, links in WCAG 2.0 AA have two minimum contrast requirements - 

- 4.5:1 contrast between the link text colour and the background.
- 3:1    contrast between the link text colour and the surrounding non-link text colour.

Which should mean the link would not pass because:

- linked text colour -> background (white default) = 3.58:1 - insufficient contrast
- linked text colour -> non-link text colour =  1.36:1 - insufficient contrast

Oddly, it correctly fails when the same colour is used for non-linked text (paragraph light color).

When you choose a more extreme contrast like #fff linked text with #fff background (so the link is completely invisible to the eye) it still passes.

Attached test case html file.

[test link colour contrast.txt](https://github.com/inclusive-design/AChecker/files/1418156/test.link.colour.contrast.txt)

**Comment by cindyli** [comment]
Thanks for the report, @ovann86. I've added this issue to the AChecker bug tracker: http://www.atutor.ca/atutor/mantis/view.php?id=5822. Will investigate at the next development cycle.
